package.json

```diff
@@ -228,6 +230,14 @@
     "webpack-isomorphic-tools": "3.0.6"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^3.4.10",
+    "@storybook/addon-console": "1.0.4",
```

Comment:
We use a caret to keep dev dependencies loose (this cuts down on greenkeeper spam). Any reason why everything you added can't have caret versions, e.g. `"^1.0.4"`?

Reply:
oops...this was unintentional. They npm module usually gives a snippet of code and I just copied that and didn't realize. I should be able to update that easily.